aws/solutions/lambda-backed-cloudformation-custom-resources/get_vpc_main_route_table_id/RouteTable.template

```diff
@@ -0,0 +1,106 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
In general, it would be worth running `cfn-lint` on this template.

Reply:
done

Reply:
done